smoke-tests/src/test/groovy/io/opentelemetry/smoketest/WebsphereSmokeTest.groovy

```diff
@@ -34,9 +34,10 @@ abstract class WebsphereSmokeTest extends AppServerTest {
   }
 }
 
-@AppServer(version = "8.5.5.20", jdk = "8-openj9")
+@AppServer(version = "8.5.5.19", jdk = "8-openj9")
```

Comment:
Why this change?

Reply:
see comment here:

https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/d7756f1f2b59ef4f4f6d0fb21a7182e5734f6f34/smoke-tests/images/servlet/build.gradle#L69-L71